syft/generic/pointers/string_pointer.py

```diff
@@ -0,0 +1,127 @@
+from typing import List
+from typing import Union
+
+from syft.generic.string import String
+from syft.generic.pointers.object_pointer import ObjectPointer
+from syft.workers.base import BaseWorker
+
+import abc
+
+
+class PointerClassMaker(abc.ABCMeta):
```

Comment:
It  strikes me that this part should be integrated in the hook file, in order to perform the hook a single time and put all the hook mechanisms at the same place.

Reply:
done